fossilize.cpp

```diff
@@ -6291,6 +6506,64 @@ static bool json_value(const VkPipelineRenderingCreateInfoKHR &create_info, Allo
 	return true;
 }
 
+template <typename Allocator>
+static bool json_value(const VkPipelineColorWriteCreateInfoEXT &create_info, Allocator &alloc, Value *out_value)
```

Comment:
One thing about ColorWriteCreateInfo in particular is that it is affected by dynamic state, e.g. VK_DYNAMIC_STATE_COLOR_WRITE_ENABLE_BIT. I think we have to ignore color write enables array when hashing.

Reply:
Yeah, I think the main problem is that we can't access the dynamic_info struct from there currently?

Reply:
Just add a TODO, and I think we can let it slide for now.

Reply:
Added a TODO.